com.github.emmanueltouzery.cigale.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Comment=
```

Comment:
```suggestion
Comment=
```
I guess you forgot a comment here

Reply:
not really. I'll just remove the Comment= line